common/views/components/BaseTabs/BaseTabs.tsx

```diff
@@ -96,9 +96,9 @@ const Tabs: FunctionComponent<Props> = ({
   }, [activeId]);
 
   function focusTabAtIndex(index: number): void {
-    (tabListRef?.current?.querySelector(
-      `#${tabs[index].id}`
-    ) as HTMLDivElement)?.focus();
+    (
+      tabListRef?.current?.querySelector(`#${tabs[index].id}`) as HTMLDivElement
+    )?.focus();
```

Comment:
nit: rather than mucking around with parens, let's use an intermediate variable:

```suggestion
    const element = tabListRef?.current?.querySelector(`#${tabs[index].id}`) as HTMLDivElement;
    element?.focus();
```